### ISSUE TITLE
Loading more than once in Render throws error

### ISSUE THREAD

**Issue by preynoldssco** [jungmannlab/picasso#5]
Render generally works well if I only process one file:

open Render
import data
export image
close Render

If I try and open more than one data file (ie open one, then open another) I get the following error:

![image](https://user-images.githubusercontent.com/12394236/38114737-668f221a-33a1-11e8-90d8-287664f147d7.png)

**Comment by straussmaximilian** [comment]
I fixed the error with the latest commit - but it adds files. Initially, the idea of `open file` was to add and keep all files in one render window (e.g., for multicolor images). I usually drag and drop so I didn't notice that we got a bug in the file menu. However, I agree that it would be beneficial that one can process one file after another. Probably I'll include an Add button and Open File button for clarity.  I keep this issue open until it's implemented.
